docs/topics/async.txt

```diff
@@ -0,0 +1,54 @@
+====================
+Asynchronous Support
+====================
```

Comment:
This doc can be backported to the Django 3.0 (except the last paragraph). Please split your changes into two commits:
- 1st, `Doc'd asynchronous support`.
- 2nd with new environment variable (`versionchanged` annotation and release notes for Django 3.1).

Reply:
The intent is to backport the entire thing into 3.0, the new environment variable included, as this is probably going to be seen as a bug by anyone using Jupyter (that's where I heard of this from).

Reply:
But we don't backport new features. Do you think it's a bug and in such case a release blocker?

Reply:
Yes, we should probably class it as a release blocker bug, then. I wouldn't have released 3.0 if I knew about this and that we didn't have a way around it.

Reply:
(Bug in new feature, I'd say.)

Reply:
Agreed.

Reply:
Still, we should split this into two commits, first with a documentation of asynchronous support (that already exists) and second with a bugfix (and release notes for Django 3.0.1).

Reply:
The documentation, though, is only for the bugfix - I'm not sure what else I could preserve other than the first paragraph in a different commit?

Happy to do the 3.0.1 release notes separately - I'll kick off a new entry just for that, without an expected date.

Reply:
@andrewgodwin 908c67e719a54b8b612cb7df732126f42350f3fa 👀 :)

Reply:
Lovely, even easier. I added in the 3.0.1 release notes to make it all a single commit, since I believe those need to be backported as well, but if there's a sensible split here I'm not seeing please say.